rootfs/scheduler/__init__.py

```diff
@@ -1053,25 +1047,39 @@ def get_secrets(self, namespace, **kwargs):
 
         return response
 
-    def create_secret(self, namespace, name, data, secret_type='Opaque', labels={}):
+    def _build_secret_manifest(self, namespace, name, data, secret_type='Opaque', labels=None):
```

Comment:
I couldn't find the source of that so I just changed it back for now. Will rather tackle this kind of change if we go strictly typed